lib/index.js

```diff
@@ -91,6 +91,10 @@ SaaSquatch.prototype.identify = function(identify) {
     }
   }
 
+  if (init.payment_provider_id === 'null') {
+    init.payment_provider_id = null;
```

Comment:
Why is a user passing `'null'` instead of `null` - and why are we cleaning it up? To me if they pass the string `'null'` we should send it as is.